Makefile

```diff
@@ -59,6 +59,10 @@ setup_env:
 	poetry run python -c "import skorch" || true # Details above
 	poetry run python -c "from brevitas.core.scaling import AccumulatorAwareParameterPreScaling" || true # Details above
 
+	# For some issues with Mac Intel
```

Comment:
if you prefer, I could
- make it done _only_ for Mac Intel
- set the brevitas version

Reply:
It would be better yes please